lib/mix/tasks/gen.managers.ex

```diff
@@ -2,7 +2,7 @@ defmodule Mix.Tasks.Gen.Managers do
   use Mix.Task
   alias Safira.Accounts
 
-  @domain "enei.pt"
+  @domain "sei.pt"
```

Comment:
The url is actually `seium.org` :link:

Reply:
okay, will change that, but are these actual emails?